### ISSUE TITLE
bug with FXForm

### ISSUE THREAD

**Issue by realcarlos** [ViccAlexander/Chameleon#83]
when set `[Chameleon setGlobalThemeUsingPrimaryColor:[UIColor flatBlueColor] withContentStyle:UIContentStyleContrast];`

`UIStatusBar` and `UITableViewCell` display the wrong color.

**Comment by bre7** [comment]
Upload a screenshot of you can

**Comment by realcarlos** [comment]
![Uploading SAU8@PHTKA$7[]TV1_$MQU0.jpg…]()

**Comment by realcarlos** [comment]
![Uploading Screen Shot 2015-11-01 at 6.55.42 PM.png…]()
![Uploading SAU8@PHTKA$7[]TV1_$MQU0.jpg…]()
![Uploading IMG_7775.PNG…]()

**Comment by realcarlos** [comment]
![screen shot 2015-11-01 at 6 55 42 pm](https://cloud.githubusercontent.com/assets/13853/10868520/90bd5cae-80cb-11e5-8266-69bf41573ccd.png)
![sau8 phtka 7 tv1_ mqu0](https://cloud.githubusercontent.com/assets/13853/10868521/968c604e-80cb-11e5-840d-7cd71b099f26.jpg)
![Uploading IMG_7775.PNG…]()

**Comment by bre7** [comment]
Which color should it be ? Keep in mind that globalTheme methods affect almost all the UI components in the app

**Comment by realcarlos** [comment]
I think that the color of checkmark should be flatBlueColor,but it becomes white.

**Comment by bre7** [comment]
That's because how UIBittons are themed.

**Comment by ViccAlexander** [comment]
> UIStatusBar and UITableViewCell display the wrong color.

If you're seeing unexpected results for `UIStatusBar`, make sure you set the following key `View controller-based status bar appearance` in your `info.plist` file to **NO**. 

With regards to the `UITableViewCell`, I have found a solution. Add these two lines to your view controller and it should fix the problem:

``` objective-c
[[UIButton appearanceWhenContainedInInstancesOfClasses:@[[UITableViewCell class]]] setBackgroundColor:ClearColor];
[[UIButton appearanceWhenContainedInInstancesOfClasses:@[[UITableViewCell class]]] setTintColor:FlatBlue];
```

The reason I haven't added this is because I haven't decided whether it should be the default behavior or not. Obviously this is really useful, but it also messes up any other button in a `UITableViewCell`. Until I figure out how to only affect these accessory view buttons and not all `UIButton's` inside `UITabelViewCell's` it'll just have to be done manually.

> and same problem occurs in UIImagePickerController ,the background color of cancel button and choose button

This has been solved in Chameleon 2.0.5. :+1:
